Dockerfile

```diff
@@ -16,3 +16,8 @@ WORKDIR /home/GraphQLApp
 
 RUN apt-get update && apt-get install -y python3.10.5 python3.10.5-dev
 
+RUN sudo apt-get -y install python3-pip
+
+RUN pip3 install --user pipenv
```

Comment:
Maybe we can combine those three layers into one?

Reply:
I merged them into one, fewer layers are better layers